src/main/java/lotto/domain/Lotto.java

```diff
@@ -1,66 +1,32 @@
 package lotto.domain;
 
-import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
-public class Lotto {
-
-    public final static int GAME_PRICE = 1000;
-
-    private final int purchaseCount;
-    private final List<LottoNumber> lottoNumbers;
-
-    public Lotto(int purchaseAmount) {
-        this.purchaseCount = getPurchaseCount(purchaseAmount);
-        this.lottoNumbers = purchaseLotto();
-    }
-
-    public MatchResult getMatchResult(LottoNumber matchLotto) {
-        List<LottoMatch> matchList = new ArrayList<>();
-        for (LottoNumber lottoNumber : lottoNumbers) {
-            matchList.add(new LottoMatch(matchLotto, lottoNumber));
-        }
-        return new MatchResult(matchList);
-    }
+import static lotto.domain.Money.GAME_PRICE;
 
-    private List<LottoNumber> purchaseLotto() {
-        return generateLotto(purchaseCount);
-    }
+public class Lotto {
```

Comment:
Lotto 클래스가 수정되면서 Rank를 보관하는 역할을 하고 있는데요.
아래와 같이 클래스 네이밍이 수정해보는건 어떨까요?
```suggestion
public class LottoResult {
```